libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1051,11 +1051,20 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 		m_code << templ.render();
 		break;
 	}
+	case FunctionType::Kind::Error:
+	{
+		solAssert(false, "");
+	}
 	case FunctionType::Kind::Assert:
 	case FunctionType::Kind::Require:
```

Comment:
Now that I think of it, none of the tests checks that you can't use errors with `assert`. Or can you?

Reply:
I'll add one.